src/source/PeerConnection/PeerConnection.c

```diff
@@ -1008,6 +1015,26 @@ STATUS restartIce(PRtcPeerConnection pPeerConnection)
     return retStatus;
 }
 
+PUBLIC_API NullableBool canTrickleIceCandidates(PRtcPeerConnection pPeerConnection)
+{
+    NullableBool canTrickle = {
```

Comment:
This is not C syntax. Please use ANSI C syntax {FALSE, FALSE}

Reply:
which c standard are we complying with? this is compliant with c99 https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html.

Reply:
I would prefer C89 whenever possible. There is no reason to deviate from it. All of the initializers are anonymous in the codebase.

Reply:
but aws-c-io and aws-c-common is only c99. Once we move to them any effort we spent on c89 is wasted.

Reply:
I don't think this is an effort or we are wasting anything. I also don't think in this case you are actually getting any advantage of the named initialization params.

Reply:
isnt it more readable?

Reply:
There are many syntactic sugars that higher languages provide but I would try to be as conservative as possible - especially since it doesn't server any purpose. There is little or none in the codebase that won't allow compiling with c89 only - at least the core.

Reply:
but once we move to aws sdk, it wont be c89 anymore.

Reply:
fixed